pom.xml

```diff
@@ -10,7 +10,7 @@
   </parent>
 
   <artifactId>anchore-container-scanner</artifactId>
-  <version>1.0.26-SNAPSHOT</version>
+  <version>1.1.0-SNAPSHOT</version>
```

Comment:
Must ensure next tag is 1.1.0 (just leaving the comment so it's clear to whoever does the tag)